Orange/widgets/tests/base.py

```diff
@@ -813,3 +813,9 @@ def data_one_column_nans(cls):
                 "ynyn"
             )))
         return table
+
+    @classmethod
+    def data_one_column_infs(cls):
+        table = cls.data_one_column_nans()
+        table.X[:, 1] = np.inf
+        return table
```

Comment:
Could extend `data_one_column_nans()` into `data_one_column_vals(value=np.nan)` and call as `data_one_column_vals(np.inf)`.

Reply:
Sure.